com.jetbrains.PhpStorm.json

```diff
@@ -153,17 +149,23 @@
             ]
         },
         {
-            "name": "wrapper",
+            "name": "ide-flatpak-wrapper",
             "buildsystem": "meson",
             "config-opts": [
                 "-Deditor_binary=/app/extra/phpstorm/bin/phpstorm",
-                "-Dprogram_name=phpstorm",
-                "-Deditor_title=PhpStorm"
+                "-Dprogram_name=phpstorm-wrapper",
+                "-Deditor_title=PhpStorm",
+                "-Dfirst_run_template=phpstorm_first_run.txt",
+                "-Dflagfile_prefix=flatpak-phpstorm"
```

Comment:
Why that ?

Reply:
Yeah but why would you want to prefix them ? They're infernal, just leave them as it is.

Reply:
I'm afraid if we change the prefix, it will show the first_run file to users that are just updating the IDE (and not launching it for the first time).

Reply:
Oh yeah, just checked, it matches the existing files this way, so it is valid indeed.

Reply:
i just copied other ides using same wrapper